server/Dockerfile

```
@@ -0,0 +1,7 @@
+FROM node
+WORKDIR /usr/app
+COPY ./types.d.ts ../
+COPY . .
+RUN  npm install typescript  -g && npm install
```

How can you not notice extra spaces? Also why are we installing typescript separately, it's already a dev dependency 
```suggestion
RUN npm install typescript -g && npm install
```

---

We are installing typescript globally for using command `tsc`